samples/42.scaleout/src/main/java/com/microsoft/bot/sample/scaleout/BlobStore.java

```diff
@@ -107,16 +115,15 @@ public BlobStore(String accountName, String accountKey, String containerName) th
             } else {
                 blob.uploadText(content);
             }
-        } catch (URISyntaxException e) {
+            return CompletableFuture.completedFuture(true);
+        } catch (URISyntaxException | IOException e) {
             e.printStackTrace();
+            return CompletableFuture.completedFuture(false);
```

Comment:
```suggestion
            return CompletableFuture.completedFuture(null);
```

The only `return false` should be in if the `HttpStatusCode` is `PRECONDITION_FAILED`, the rest of the exceptions it can be `null`.. this is something we need to validate